ink2pdf

```diff
@@ -76,6 +76,16 @@ if ($BGLAYER) {
     }
 }
 
+# Verbosity should affect inkscape stderr, too
+my $inkscape_stderr = $VERBOSE == 2 ? "" : "2>/dev/null";
+
+# Option for PDF output was changed in 1.x
+my $inkscape_version = `inkscape --version $inkscape_stderr`;
+$inkscape_version =~ /(\d+)\.(\d+)\.(\d+)/
+    or die "*** Cannot determine inkscape version from '$inkscape_version'";
+print $inkscape_version if $VERBOSE == 2;
```

Comment:
Let's use Perl's standard version module to compare versions, since we don't control Inkscape's versioning scheme (they could come up with a future "1.3_40" version for example - who knows!).  

At the top, add a `use version;` then compare like:
```
    $inkscape_version =~ /v?(\d+)([\d\._]+)/i
        or die ...;
    my is_inkv1 = version->parse($inkscape_version) >= "v1.0.0";
```
The version regex is a suggestion, anything looser will do.

Reply:
@TpmKranz I merged your changes -- thanx!  I'll apply my suggestions above in the main line.